### ISSUE TITLE
Add support firebase-admin 12 in peerDependencies

### ISSUE THREAD

**Issue by mcfarljw** [firebase/firebase-functions-test#216]
https://github.com/firebase/firebase-admin-node has been updated to `v12` and is not a version included in the current `peerDependencies`.

```
"firebase-admin": "^8.0.0 || ^9.0.0 || ^10.0.0 || ^11.0.0 || ^12.0.0",
```

**Comment by mcfarljw** [comment]
Related to https://github.com/firebase/firebase-functions/issues/1496.

**Comment by tonprince** [comment]
I have the same issue when trying to use the latest version of firebase-admin with firebase-functions.

**Comment by willbattel** [comment]
Hey @taeold when you have a chance could you please look into this or let us know who we should reach out to? We cannot update `firebase-admin` to `12.0.0` until we get an update for `firebase-functions-test`.

Also, just want to check if this lib is still supported? I noticed that there hasn't been any activity in the last 9 months.

Thanks!

**Comment by Hibrix-net** [comment]
Same here, please help.

**Comment by nternouski** [comment]
Same, please fix it

**Comment by mcfarljw** [comment]
It looks like this was just fixed in `3.1.1`
